scss/window/_theme.scss

```diff
@@ -1,24 +1,37 @@
-// TODO: extract or remove modal variables
-$modal-content-bg: $widget-bg !default;
-$modal-content-xs-box-shadow: 0 3px 9px rgba( 0, 0, 0, .5 ) !default;
-$modal-header-border-color: transparent !default;
-
 @include exports("window/theme") {
 
-    .k-window {
-        @include box-shadow($modal-content-xs-box-shadow);
-        background-color: $modal-content-bg;
+    $window-shadow: 0 3px 3px 0 rgba(0, 0, 0, .06) !default;
+    $window-focused-shadow: 1px 1px 7px 1px rgba(0, 0, 0, .3) !default;
 
-        .k-window-action {
-            opacity: 1;
 
-            .k-icon:hover {
-                color: $accent-contrast;
-            }
-        }
+    .k-window {
+        @include appearance( panel );
+        border-width: 0;
+        box-shadow: $window-shadow;
+    }
+    .k-window.k-state-focused {
+        box-shadow: $window-focused-shadow;
     }
 
+
+    // Titlebar
     .k-window-titlebar {
-        border-color: $modal-header-border-color;
+        @include appearance( header );
+        border-color: inherit;
+        background-clip: padding-box;
     }
-}
+
+
+    // Actions
+    .k-window-action {
+        opacity: 1;
+        transition: transform .1s ease-in-out;
+
+        &:hover,
+        &.k-state-hover {
+            opacity: 1;
+            transform: scale(2);
```

Comment:
It's not ideal, to say the least. After a discussion with @yordanov, the close button seems to be a good candidate for a `.k-bare` button, without the scaling.